node_modules/oae-core/manageaccess/manageaccess.html

```diff
@@ -24,7 +24,11 @@ <h3 id="manageaccess-modal-title">__MSG__MANAGE_ACCESS__</h3>
                             <span id="manageaccess-overview-shared-title"><!-- --></span>
                         </h4>
                         <div id="manageaccess-overview-shared" class="well">
-                            <ul id="manageaccess-overview-selected" class="oae-list oae-list-compact"><!-- --></ul>
+                            <ul id="manageaccess-overview-selected-shared-container" class="oae-list oae-list-compact manageaccess-overview-selected"><!-- --></ul>
+                        </div>
+                        <div id="manageaccess-overview-invitations" class="well hide">
```

Comment:
The responsive version of this widget no longer fully works (FF 40)

<img width="373" alt="screen shot 2015-09-13 at 17 31 25" src="https://cloud.githubusercontent.com/assets/109850/9837643/cfb6eb2a-59fa-11e5-9243-17d2d2bae58e.png">